### ISSUE TITLE
Icons

### ISSUE THREAD

**Issue by cancng** [Satoqz/vscode-discord#35]
Changing the language icon is possible? I think js, ts, php icons are not good. Can we customize icons 😊

**Comment by Satoqz** [comment]
Hey there,

I can understand your point and i will look into possible alternatives for these icons :)

Technically you can customize them, yes!
It would require you to create an application on the discord developer portal and upload your icons with the same key names that i used (just look into the code, easy to find). Then you can copy the public client id of your app and use it in the extension config which includes an option to use a custom id.

So in conclusion, yes its customizable, but a lot of work.

It would probably be easier if you could maybe suggest alternative icons if you have a clear idea. I'm very welcoming to suggestions :)

**Comment by cancng** [comment]
Thanks for the reply. I can recommend this repo for icons. But it doesn't have JSX, TSX, etc. icons but your extension already have 👍 

https://github.com/MrSquaare/icongram

**Comment by Satoqz** [comment]
Actually, I really like the icons you that you suggested. They keep the simplicity but add more color variation. I created a fork of the repository and will add some of the missing icons that are required in my opinion trying to stick with the style and later update the extension :+1:
